images/chromium/config/main.tf

```diff
@@ -0,0 +1,50 @@
+terraform {
+  required_providers {
+    apko = { source = "chainguard-dev/apko" }
+  }
+}
+
+variable "extra_packages" {
+  description = "The additional packages to install (e.g. chromium)."
+  default     = ["chromium"]
+}
+
+variable "environment" {
+  default = {}
+}
+
+module "accts" {
+  source = "../../../tflib/accts"
+  run-as = 65532
+  uid    = 65532
+  gid    = 65532
+  name   = "chrome"
+}
+
+output "config" {
+  value = jsonencode({
+    archs = ["x86_64"]
+    contents = {
+      packages = concat([
+        "busybox",
```

Comment:
Do we need these?

Reply:
Both are used by the wrapper

Reply:
I should probably move these over to the package as runtime deps